Contributing.md

```diff
@@ -166,19 +153,60 @@ Once your changes are successfully merged into the main repository, delete the b
 
 Congratulations, you have successfully contributed to the project!
 
-## How to use Markdown to format your topic
+## Adding samples
+
+Samples are a critical tool for learning the Office Scripts API. We welcome samples scripts from the community to help demonstrate useful scripting scenarios.
+
+### Adding a sample script to the GitHub repository
+
+All the example code in this repository comes from one of the base sample **.yaml** files. The Office Scripts for Excel samples are in **[/docs/sample-scripts/excel-scripts.yaml](/docs/sample-scripts/excel-scripts.yaml)**. Use one of the methods described earlier in this guide to edit the file. You can also create an issue with the desired sample code and we'll add it to the documentation.
+
+Samples are tied to particular APIs. Pick the API that your sample showcases and use it as the YAML key. A YAML key for the Excel sample file has this format: `'ExcelScript.'`*class-name*`#`*method-name*`:member(`*overload-number*`)`. The *overload-number* indicates which method is being referenced when their are multiple options. The value is usually `1`.
+
+After the key, add `:`, then add `  - |-` on a new line. This ensures the spacing in the example stays as is.
```

Comment:
I got a little confused reading this -- probably because I'm not very familiar with Office Scripts -- so I suggest specifying the type of key here. 

```suggestion
After the YAML key, add `:`, then add `  - |-` on a new line. This ensures the spacing in the example stays as is.
```